.gitignore

```diff
@@ -1 +0,0 @@
-go.sum
```

Comment:
You can fill this file with output from [this website](https://www.toptal.com/developers/gitignore/).